tests/lms/extensions/feature_flags/_providers_test.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+from lms.extensions.feature_flags import config_file_provider
+from lms.extensions.feature_flags import envvar_provider
+from lms.extensions.feature_flags import query_string_provider
+
+
+class TestConfigFileProvider:
+    @pytest.mark.parametrize(
+        "settings,result",
+        [
+            ({"feature_flags.test_feature": "true"}, True),
+            ({"feature_flags.test_feature": "false"}, False),
+            ({}, None),
+        ],
+    )
+    def test_it(self, pyramid_request, settings, result):
+        pyramid_request.registry.settings.update(settings)
+        assert config_file_provider(pyramid_request, "test_feature") == result
+
+
+class TestEnvVarProvider:
+    @pytest.mark.parametrize(
+        "envvars,result",
+        [
+            ({"FEATURE_FLAG_TEST_FEATURE": "true"}, True),
+            ({"FEATURE_FLAG_TEST_FEATURE": "false"}, False),
```

Comment:
Are there other representations of `True` and `False` in env vars that we might want to test here?

Reply:
Naah. Technically anything that Pyramid considers to be a [truthy string](https://docs.pylonsproject.org/projects/pyramid/en/latest/glossary.html#term-truthy-string) will work, but `true` is the only thing officially supported by this extension's docs